ui/app/rif/components/modal/addNewTokenNetworkAddress/addNewTokenNetworkAddress.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Select from 'react-select'
+
+class AddNewTokenNetworkAddress extends Component {
+	static propTypes = {
+		updateNetwork: PropTypes.func.isRequired,
+		updateAddress: PropTypes.func.isRequired,
+		networks: PropTypes.array.isRequired,
+	}
+    constructor(props) {
+        super(props);
+		let networks = []
+		Object.assign(networks, props.networks)
```

Comment:
This can be done in one line, and also networks can be a constant since it will not be modified on this scope

`const networks = Object.assign({}, props.networks)`

or just use the spread operator:

 `const networks = ...props.networks`

Reply:
Fixed in next commit